README.md

```diff
@@ -280,6 +101,4 @@ I would like to thank Microverse.
 
 This project is [MIT](./LICENSE) licensed.
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] The License link in the README file is broken, Please make sure to fix this. It will make the README file more professional.

Reply:
well resolved! ✔️